### PR TITLE
Hello world bar

### PR DESCRIPTION
**Pull request by ravitb**

### CODE REVIEW

File: docs/quick-start/hi-kin-node.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ We can now create a KinAccount object, we will use it to interact with our accou
 This is the app ID of our account: anon
 ```
 
-Details of the `friendbot` service are too detailed for our Hello World tutorial, so when you're ready you should read [this](../documentation/nodejsjs-sdk#friendbot).
+Details of the `friendbot` service are too detailed for our Hello World tutorial, so when you're ready you should read [this](../documentation/nodejs-sdk#friendbot).
```
Comment (cnotethegr8) [inline docs/quick-start/hi-kin-node.md:102]: outdated link

File: docs/quick-start/hi-kin-node.md [hunk 2]
```diff
@@ -212,15 +212,14 @@ Now that you have a destination public address, you can send Kin to the associat
 The transaction succeeded with the hash  251fa6e5835d49c6bfddb904cded853285177a96a1d625fd368d87a75b2c01c2
 ```
 
-Not all transactions executed on the blockchain will be charged a fee. To learn more about transaction fees and whitelisting, see [Whitelist](../documentation/node-sdk#transferring-kin-to-another-account-using-whitelist-service).
+Not all transactions executed on the blockchain will be charged a fee. To learn more about transaction fees and whitelisting, see [Whitelist](
+.
```
Comment (cnotethegr8) [inline docs/quick-start/hi-kin-node.md:216]: the link is removed here. you have an open ( with no link and closer

File: docs/quick-start/hi-kin-node.md [hunk 3]
```diff
@@ -212,15 +212,14 @@ Now that you have a destination public address, you can send Kin to the associat
...
 ## Conclusions
 This was a very short introduction to the Kin SDK for node. This SDK is meant to run on a server and be between your client apps and the Kin Blockchain.
 The SDK provides plenty more features for you to explore including support for channels to maximize your transaction throughput.
 
-[//]: # (## Downloads)
 
-[//]: # (Download the full helloWorld.js for your convenience.)
-
-[//]: # (Track progress and download the Node SDK on GitHub.)
+[Download the full helloWorld.js for your convenience.](/kin-website-docs/hello-world/kin-sdk-node-hello-world.zip).
```
Comment (cnotethegr8) [inline docs/quick-start/hi-kin-node.md:224]: outdated link

File: website/sidebars.json [hunk 4]
```diff
@@ -30,6 +30,7 @@
       "documentation/python-bootstrap"
     ],
     "NodeJs": [
+      "quick-start/hi-kin-node",
```
Comment (cnotethegr8) [inline website/sidebars.json:33]: outdated link

**Review by cnotethegr8 — CHANGES_REQUESTED** [review]

**Comment by yosriz** [comment]
already merged with https://github.com/kinecosystem/kin-website-docs/pull/55

**Pull request closed without merging**